modules/gatsby/src/components/github/github-contributors.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+
+export default class GithubContributors extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      response: window.fetchContributorsResponse
+    };
+  }
+
+  componentDidMount() {
+    if (window.fetchContributorsResponse) return;
+
+    const {project} = this.props;
+    fetch(`https://api.github.com/repos/${project}/contributors`)
+      .then(response => response.json())
+      .then(response => {
+        window.fetchContributorsResponse = response;
```

Comment:
Why is this a global variable? Is this class being mounted and unmounted? Add a comment?

Can we declare a "local" global variable inside this file (or store the response on the `GithubContributors` class as static field rather than referencing `window` directly?

Maybe add a utility file/functions to query and cache github data?